### PR TITLE
Feature/nm 18

### PR DESCRIPTION
**Pull request by amolcan**
Get comments count in pull requests for specific user in all of the repositories a user is in.

### CODE REVIEW

File: services/comments.service.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const utils = require('./utils.service');
+const githubService = require('./github.service');
+const axios = require('axios');
+
+function getCommentsPerRepositoryForUser(repositoryName) {
```
Comment (tjalc) [inline services/comments.service.js]: could you please move this function to github.service?

Comment (ajancus) [reply services/comments.service.js]: Also dont update response there. It is possible to add date range (also validate Dates in controller)?

Comment (amolcan) [reply services/comments.service.js]: Requested changes done.

File: services/github.service.js [hunk 2]
```diff
@@ -128,12 +128,22 @@ function getCommitsForRepositoryPerUser(userName, repositoryName, startDate, end
...
 module.exports = {
     getCommitsUrls,
     getCommitDetails,
     getGithubDataForTeam,
     getGithubDataForUsername,
     getRepositoryContributors,
+    getCommitsForRepository,
     getCommitsForRepositoryPerUser,
-    getCommitsForRepository
+    isEventCreatedBetween,
```
Comment (ajancus) [inline services/github.service.js]: This method isEventCreatedBetween should be moved to utils.service

Comment (amolcan) [reply services/github.service.js]: I will move it to utils.service, as another method in github.service is using it already.

Comment (ajancus) [reply services/github.service.js]: Testing endpoint which i deleted was using it

Comment (amolcan) [reply services/github.service.js]: Done, method moved to utils.service class.

File: services/comments.service.js (hunk 1 above)
Comment (amolcan) [reply services/comments.service.js]: Added dates validation in comments.controller class.

File: controllers/comments.controller.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    utils.validateDates(startDate, endDate);
+
+    if (!userName) {
+        throw  new Error(`Missing required parameter: userName = ${userName}. `);
+    }
+
+    let comments = commentsService.getCommentsCountForUserName(userName, startDate, endDate);
+
+    console.log(comments);
```
Comment (SilviaKorcekova) [inline controllers/comments.controller.js]: please remove console.log :)

File: services/github.service.js [hunk 4]
```diff
@@ -128,12 +111,21 @@ function getCommitsForRepositoryPerUser(userName, repositoryName, startDate, end
         });
 }
 
+function getCommentsPerRepositoryForUser(repositoryName) {
```
Comment (SilviaKorcekova) [inline services/github.service.js]: please rewrite this function to match other already rewrited functions for api call :)

**Pull request merged**